test/parallel/test-readline-interface.js

```diff
@@ -175,6 +175,57 @@ function isWarned(emitter) {
   assert.equal(callCount, expectedLines.length);
   rli.close();
 
+  // \t when there is no completer function should behave like an ordinary 
+  //   character
+  fi = new FakeInput();
+  rli = new readline.Interface({ input: fi, output: fi, terminal: true });
+  called = false;
+  rli.on('line', function(line) {
+    assert.equal(line, '\t');
+    called = true;
```

Comment:
Probably a good idea to change all my instances of `assert.ok()` and `assert.equal()` to `.strictEqual()` since I know exactly what the value and type should be, so I'll go ahead and do that. Stop me if that's a terrible idea for some reason I'm not considering.